database/java/2017/3163.yaml

```diff
@@ -0,0 +1,22 @@
+cve: 2017-3163
+title: "Apache Solr ReplicationHandler path traversal attack"
+description: >
+    When using the Index Replication feature, Apache Solr nodes can pull index files from a master/leader node using an HTTP API which accepts a file name. However, Solr before 5.5.4 and 6.x before 6.4.1 did not validate the file name, hence it was possible to craft a special request involving path traversal, leaving any file readable to the Solr server process exposed. Solr servers protected and restricted by firewall rules and/or authentication would not be at risk since only trusted clients and users would gain direct HTTP access
+cvss_v2: 5
+references:
+    - https://nvd.nist.gov/vuln/detail/CVE-2017-3163
+    - http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-3163
+    - http://www.cvedetails.com/cve/CVE-2017-3163
+    - https://issues.apache.org/jira/browse/SOLR-10031
+affected:
+    - groupId: "org.apache.solr"
+      artifactId: "solr-core"
+      version:
+        - ">=1.4.0,1.4"
```

Comment:
I wasn't able to validate 1.x from https://nvd.nist.gov/vuln/detail/CVE-2017-3163/cpes?expandCpeRanges=true. Can you double check?

Reply:
@ashcrow Im not able to get a direct link, see (here)[http://lucene.apache.org/solr/news.html], just find the cve in the page (under title "15 February 2017 - [SECURITY] CVE-2017-3163")